unity-client/Assets/Tutorial/Prefabs/Common/TutorialRobot/Materials/Black_MTL.mat

```diff
@@ -9,17 +9,21 @@ Material:
   m_PrefabAsset: {fileID: 0}
   m_Name: Black_MTL
   m_Shader: {fileID: 4800000, guid: 7bc4dc8666ba3c94e80873a8e72e5b06, type: 3}
-  m_ShaderKeywords: VERTEX_COLOR_ON _EMISSION _NORMALMAP
-  m_LightmapFlags: 1
-  m_EnableInstancingVariants: 1
+  m_ShaderKeywords: 
+  m_LightmapFlags: 6
+  m_EnableInstancingVariants: 0
   m_DoubleSidedGI: 0
-  m_CustomRenderQueue: 2302
```

Comment:
As we are changing the `renderQueue` here, please make sure we aren't breaking anything with this change.

Reply:
I did test it and it seems to work as expected. Probably we should wait for @sandrade-dcl to test this as well since he knows better the onboarding flow 😅

Reply:
I have been testing and the tutorial is working fine, but it seems the flow of the "GoToGenesis" taskbar icon was broken (I don't know why). I just have commited the fix and now it is working fine.